extensions/amp-animation/0.1/web-animations.js

```diff
@@ -541,6 +552,16 @@ export class MeasureScanner extends Scanner {
   }
 
   /** @override */
+  onSwitchAnimation(spec) {
+    // The first to match will be used; the rest will be ignored.
+    this.with_(spec, () => {
+      spec.switch.some(candidate => {
```

Comment:
took me a bit to realize `some` is used to short-circuit a loop. Is the short-circuiting behavior guaranteed in every implementation? honestly a for-loop might be more readable.